region/lowernorfair/west/Golden Torizo's Room.json

```diff
@@ -261,11 +252,20 @@
                 "Ice",
                 "Wave",
                 "Plasma",
-                {"enemyDamage": {
-                  "enemy": "Golden Torizo",
-                  "type": "contact",
-                  "hits": 4
-                }}
```

Comment:
Without using the safe spot, I think that all of these beam kills need at least 1 hit, even with `canDodgeWhileShooting`. The fight is hard to do damageless and Samus could come in with low energy. It often takes a while to get to the first farm cycle, and while tankless+suitless, farming doesnt help, as 1 hit will kill Samus.

Reply:
Maybe the line of thought should be that Safe Spot is always better, so these strats are for those who don't know about it?  
And dodging GT isn't that hard, but doing it for the whole fight is.  And not getting hit by projectiles at all is pretty tough but you can usually farm that health back.  However that would come out of your heat frame pool so healing off extra damage shouldn't count.